books/free-programming-books-subjects.md

```diff
@@ -262,6 +262,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 ### Game Development
 
 * [2D Game Development: From Zero To Hero](https://github.com/Penaz91/2DGD_F0TH) - Daniele Penazzo (HTML, [PDF, EBPUB, Kindle...](https://therealpenaz91.itch.io/2dgd-f0th#download)) (:construction: *in process*)
+* [3D Math Primer for Graphics and Game Development](https://gamemath.com/book/intro.html) - Fletcher Dunn (HTML)
```

Comment:
Should this use top url?

https://gamemath.com/book

Reply:
no